clients/gui-app/src/lib/host-rpc-policy/host-method-policy-table.ts

```
@@ -588,6 +591,11 @@ export const HOST_METHOD_POLL_TABLE = {
     joinResponseTimeoutMs: null,
     poll: null,
   },
+  "worktree.setRepoScripts": {
+    mode: "fifo",
+    joinResponseTimeoutMs: null,
+    poll: null,
+  },
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)